amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -628,33 +697,33 @@ Removes a permission from this message queue.
 |Type |String
```

Comment:
Same applies here, String returned by this operation contains ID of the request, shouldn't we mention it here?